src/nodeHelper.ts

```diff
@@ -219,6 +220,16 @@ export class NodeHelper {
         return NodeHelper.getExtensionsProperty(op, NodeHelper.SPLIT_OPERATION_ORIGINAL_OPERATION, null);
     }
 
+    public static addSplitOperationIntoOperations(op: Operation, splittedOp: Operation) {
```

Comment:
how about make it more generic to have it in NodeHelper, we dont need split concept here, for example, addExtendedOperation(node: Operation, operation: Operation)

Reply:
I am worried about generic may cause confusing. Because previously we put these operations in cli-operations, which makes az cannot know these operations come from poly or split. That's why I create an independent property for split.

Reply:
Removed reference. Thanks